HW4/MykolyshynV/4.3

```diff
@@ -0,0 +1,12 @@
+input_num = int(input())
```

Comment:
n=int(input('enter some number from fibonacci list='))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n}, {fibonacci_numbers}')